src/main/java/org/folio/inventory/resources/AbstractInstances.java

```diff
@@ -70,11 +71,11 @@ public AbstractInstances(final Storage storage, final HttpClient client) {
     CollectionResourceClient relatedInstancesClient = createInstanceRelationshipsClient(
       routingContext, context);
     CollectionResourceRepository relatedInstancesRepository = new CollectionResourceRepository(relatedInstancesClient);
-    List<String> instanceId = Arrays.asList(instance.getId());
+    List<String> instanceId = Collections.singletonList( instance.getId() );
     String query = createQueryForRelatedInstances(instanceId);
 
     CompletableFuture<Response> future = new CompletableFuture<>();
-    relatedInstancesClient.getMany(query, future::complete);
+    relatedInstancesClient.getMany(query, Integer.MAX_VALUE, 0, future::complete);
```

Comment:
The magic values `Integer.MAX_VALUE, 0` should be avoided by adding a method like getAll(String query, Consumer<Response>) to CollectionResourceClient. This also avoids duplication of this code in Instances.java.

Javadoc should be added to CollectionResourceClient.getMany pointing out the most APIs be default have a limit of 10 records.

Reply:
Would both of those recommendations apply to MultipleRecordsFetchClient.java, which also uses magic values and has no Javadoc at all.

Reply:
> The magic values `Integer.MAX_VALUE, 0` should be avoided by adding a method like getAll(String query, Consumer) to CollectionResourceClient. This also avoids duplication of this code in Instances.java.

I like the idea of creating a getAll method to make the intent to get all of the records explicit.

> Javadoc should be added to CollectionResourceClient.getMany pointing out the most APIs be default have a limit of 10 records.

I don't understand the use of Javadoc for this. I agree with the idea of a comment that explains why this uses a specific value for the limit to get all records.

Reply:
The underlying cause of the bug that this PR fixes is that many APIs have a default limit of 10.
Adding javadoc to getMany(String, Consumer) that warns about this default limit might prevent making this bug again.